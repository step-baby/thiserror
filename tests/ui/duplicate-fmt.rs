use thiserror::EnumDisplay;

#[derive(EnumDisplay, Debug)]
#[display("...")]
#[display("...")]
pub struct Error;

fn main() {}
