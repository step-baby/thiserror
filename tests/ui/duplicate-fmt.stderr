error: only one #[display(...)] attribute is allowed
 --> tests/ui/duplicate-fmt.rs:5:1
  |
5 | #[display("...")]
  | ^^^^^^^^^^^^^^^
